packages/web/src/js/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const NAME_ARIA_EXPANDED = 'aria-expanded';
```

Comment:
Please use `_ATTRIBUTE` suffix instead of `NAME_` prefix, eg. `ARIA_EXPANDED_ATTRIBUTE`. I am using this kind of naming in all other files, I think. So refactor will be smoother.